👌 IMPROVE: Focus Frame reposition

Toxicom opened the pull request:
# Summary of Changes
1. Move the Focus frame below Target frame **for DPS layout**
2. Move the Focus frame above Target frame **for Healer layout**
3. Disable buffs for the Focus frame
4. Remove raw hp text, leave only percent hp
6. Change gradient direction
7. Reduce BigWigs emphasized bar limit to 4
   - On healer layout it would clip with focus in those rare occasions where you might have >4 bars

# Description
- The old focus frame position (centered below WAs) was overlapping with WAs and overall wasn't satisfied with that position
- I'm also thinking of maybe changing the way we show party/focus health and increase the font size, move it more into the frame since that's crucial info and doesn't take up much space

# Screenshots
![image](https://user-images.githubusercontent.com/69549795/208306241-5e4e03ad-dfa4-4894-b9ac-b124bdf4c1d4.png)

# To test

- [ ] `git checkout focus-frame`
- [ ] Run `/tx install` with DPS layout
- [ ] Test new focus frame and see if no overlaps etc.

Toxicom committed:
👌 IMPROVE: Clean up and move focus frame for DPS

RyadaProductions commented:
Code looks good, but any particular reason it is anchored and placed below the player unit frame and not under the current target unit frame?

Toxicom commented:
Hmm.. I guess it depends on the player. Personally I only ever use focus frame for healer/tank, never for enemy.

RyadaProductions commented:
I know a few people that use it for kicks with macro's and tanks that use it for taunt macro's, would it be an option to maybe set it as a anchor flag in the TX settings? So that people can easily anchor it to the enemy or player unit frame depending on the setting?

Toxicom committed:
👌 IMPROVE: Move focus below target for DPS

Toxicom commented:
Moved focus below target

Toxicom committed:
👌 IMPROVE: Focus frame

-   Move healer frame above target;
-   Fix custom text positions;
-   Reduce emphasized bar limit for big wigs.
